packages/components/src/components/checkbox/checkbox.jsx

```diff
@@ -21,18 +21,14 @@ const Checkbox = React.forwardRef(
         },
         ref
     ) => {
-        const [checked, setChecked] = React.useState(defaultChecked || value);
+        const [checked, setChecked] = React.useState(defaultChecked);
```

Comment:
@artsemkavaleu-deriv  Please don't change the components package code, that's reusable code.

Reply:
sure, thx for notice!